example_project/README.md

```diff
@@ -0,0 +1,8 @@
+To start project you need to:
+ - create virtualenv with: virtualenv venv
+ - activate env with: source venv/bin/activate
+ - install requirement with: pip install -r requirements.txt
+ - call migrations: ./manage.py migrate
+ - run dev server: ./manage.py runserver
```

Comment:
this starts the django runserver where I can add pages etc. how can I access the react app?